src/webgl/p5.Shader.js

```diff
@@ -563,9 +563,11 @@ p5.Shader.prototype.enableAttrib = function(
     const loc = attr.location;
     if (loc !== -1) {
       const gl = this._renderer.GL;
-      if (!attr.enabled) {
```

Comment:
Makes sense, we don't need this any more per shader attribute, now that we have a global one.